pkg/util/kubernetes/apiserver/controllers.go

```diff
@@ -44,10 +44,6 @@ var controllerCatalog = map[controllerName]controllerFuncs{
 		},
 		startAutoscalersController,
 	},
-	servicesController: {
```

Comment:
I suppose that we never needed this, it may be a leftover from last time. 
But just to make sure 
cc. @xlucas